client/js/widgets/button.js

```diff
@@ -85,10 +85,11 @@ export class Button extends Widget {
 
 
       if(a.func == 'CLICK') {
-        setDefaults(a, { collection: 'DEFAULT' });
+        setDefaults(a, { collection: 'DEFAULT', count: 1 });
         if(isValidCollection(a.collection))
           for(const w of collections[a.collection])
-            w.click();
+            for(let i=0; i<a.count; ++i)
+              w.click();
```

Comment:
I would rather click all items in the collection, and then repeat until count has been reached.

Reply:
I didn't even think about that. But I agree.

* [ ] change click order